Packing.cmake

```diff
@@ -0,0 +1,35 @@
+set(CPACK_PACKAGE_NAME "dragonflydb"
+    CACHE STRING "dragonflydb"
+)
+
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "A modern replacement for Redis and Memcached"
+    CACHE STRING "A modern replacement for Redis and Memcached"
+)
+
+set(CPACK_PACKAGE_VENDOR "DragonflyDB")
+
+set(CPACK_PACKAGE_INSTALL_DIRECTORY "/usr/bin/")
```

Comment:
It install it right there, I was making it explicit so it can be changed later

That var sets the install directory as described here https://cmake.org/cmake/help/latest/module/CPack.html
I can change it to /usr/local/bin

Reply:
So here's something strange, 

This doc page https://cmake.org/cmake/help/latest/module/CPack.html lists the var ```CPACK_PACKAGE_INSTALL_DIRECTORY``` to set the isntall dir, but in practice that var doesn't actually work
However, ```CPACK_PACKAGING_INSTALL_DIRECTORY``` does work for that exact purpose, but isn't listed on that doc page